chapters/operatorsandexpressions.tex

```diff
@@ -433,6 +433,13 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
 Expands into \lstinline!noEvent(if $v$ >= 0 then $v$ else -$v$)!.
 Argument $v$ needs to be an \lstinline!Integer! or \lstinline!Real! expression.
 \end{semantics}
+\begin{nonnormative}
+The reason this does not generate events is that in many cases that would introduce more problems than the event would solve.
+A typical case is a flow equation of the form \lstinline!abs(x)*x=y!.
+If we have to solve for \lstinline!x! and had events for \lstinline!abs(x)! we would not find a solution if \lstinline!y! passes through zero.
+
+In inverted form this equation is \lstinline!x=sign(y)*sqrt(abs(y))!, with similar issues.
```

Comment:
White-space and more explicit remark:
```suggestion
In inverted form this equation is \lstinline!x = sign(y) * sqrt(abs(y))!.
With event generation, the call to \lstinline!sqrt! would fail when applied to a negative number during root finding of the zero crossing for \lstinline!abs(y)!, compare \cref{events-and-synchronization}.
Without event generation, on the other hand, evaluating \lstinline!sqrt(abs(y))! will never fail.
```